Documentation/ApiOverview/TSFE/Index.rst

```diff
@@ -0,0 +1,216 @@
+.. include:: /Includes.rst.txt
+.. index:: TSFE; TypoScriptFrontendController
+.. _tsfe:
+
+====
+TSFE
```

Comment:
I think "TypoScriptFrontendController (TSFE)" is very long. I am thinking about the menu ... Otherwise it would be better to use the long form, I think.

Reply:
In our new Rendering Container we will be able to use different links for the menu and the main title. This is not possible yet

Reply:
For now I would suggest to go with the long title. After switching the renderings we should generally overhaul the menu tree